src/index.html

```diff
@@ -21,7 +24,375 @@
+          </h2>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="iMac"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="iMac"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
```

Comment:
All product cards have the same content, including the image, title, product code, reviews, and price. This might be placeholder content and should be updated with unique data for each product.